### PR TITLE
Support Python {3.9, 3.10, 3.11}

### PR DESCRIPTION
**Pull request by cqc-alec**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket ~= 1.10",
+        "pytket == 1.11.0rc0",
```
Comment (cqc-melf) [inline setup.py:47]: The CI should pick that up anyway, no need to update this here?

Comment (cqc-alec) [reply setup.py:47]: Ah yes you are right.

Comment (cqc-alec) [reply setup.py]: Actually it seems not in this case -- there is _no_ matching pytket 3.11 wheel for the initial install, so the build fails.

Comment (cqc-melf) [reply setup.py:47]: Oh, okay.

**Comment by cqc-melf** [comment]
Is this now ready for review?

**Review by cqc-melf — APPROVED** [review]

**Comment by cqc-alec** [comment]
> Is this now ready for review?

Sorry, yes.

**Pull request merged**